vendor/github.com/openshift/api/image/v1/types.go

```diff
@@ -417,6 +417,43 @@ type DockerImageReference struct {
 	ID string `protobuf:"bytes,5,opt,name=iD"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageStreamLayers describes information about the layers referenced by images in this
+// image stream.
+type ImageStreamLayers struct {
```

Comment:
do our generators handle this or do they produce `imagestreamlayerses`?

Reply:
talked offline... non-standard subresources get manually written expansion methods